lib/sidekiq-cron.rb

```diff
@@ -1,2 +1,7 @@
 require "sidekiq"
 require "sidekiq/cron"
+require "sidekiq/cron/job"
+require "sidekiq/cron/namespace"
+require "sidekiq/cron/poller"
+require "sidekiq/cron/launcher"
+require "sidekiq/cron/schedule_loader"
```

Comment:
@markets Please verify this change.

Reply:
What do you mean by verify 🤔? Could you please verify it by yourself in your apps too?

Reply:
It works fine with my app.